cpp/tests/c_api/create_sg_graph_test.c

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright (c) 2021, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "c_test_utils.h" /* RUN_TEST */
+
+#include <cugraph_c/cugraph_api.h>
+
+/*
+ * Simple check of creating a graph from a COO on device memory.
+ */
+int test_create_sg_graph_simple()
+{
+  int test_failed = 0;
+
+  cugraph_graph_envelope_t* G = NULL;
+  cugraph_raft_handle_t handle;
+  cugraph_device_buffer_t* src_ptr     = NULL;
+  cugraph_device_buffer_t* dst_ptr     = NULL;
+  cugraph_device_buffer_t* weights_ptr = NULL;
+  size_t num_verts                     = 4;
+  size_t num_edges                     = 3;
+  bool_t do_expensive_check            = 0;
+  bool_t store_transposed              = 0;
+  bool_t is_symmetric                  = 0;
+  bool_t is_multigraph                 = 0;
+
+  /*
+   * FIXME: populate GPU memory with a small (4 verts, 3 edges) graph COO.
+   * FIXME: return success until this test is finished.
+   */
+  return 0;
+
+  G = cugraph_make_sg_graph(&handle,
```

Comment:
I just realized I named the test and test file with the word "create" instead of "make".  Similar to Chuck's comment re: func and file names matching, I'd make these match too (either change the API to "create" or the file and test to "make").